arm-network/2016-09-01/swagger/virtualNetworkGateway.json

```diff
@@ -1075,6 +1218,105 @@
         }
       }
     },
+    "BgpPeerStatus": {
+      "properties": {
+        "localAddress": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The virtual network gateway's local address"
+        },
+        "neighbor": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The remote BGP peer"
+        },
+        "asn": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "The autonomous systen number of the remote BGP peer"
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The BGP peer state",
+          "enum": [
+            "Unknown",
+            "Stopped",
+            "Idle",
+            "Connecting",
+            "Connected"
+          ],
+          "x-ms-enum": {
+            "name": "BgpPeerState",
+            "modelAsString": true
+          }
+        },
+        "connectedDuration": {
+          "type": "string",
+          "readOnly": true,
+          "description": "How long the peering has been up"
```

Comment:
The returned string is what's given by TimeSpan.ToString in C#. However ExpressRoute uses a different format so we're keeping it as a generic string for now, in case we want to converge the APIs in the future.